.gitignore

```diff
@@ -0,0 +1,6 @@
+*.egg-info
```

Comment:
We should add IDE-kinds of files, like those in the [platform](https://github.com/edx/edx-platform/blob/master/.gitignore#L13-L21).